### PR TITLE
Mention ExtensionImages in NEWS

### PR DESCRIPTION
**Pull request by bluca**

### CODE REVIEW

File: src/shared/extension-release.c [hunk 1]
```diff
@@ -82,8 +82,13 @@ int parse_env_extension_hierarchies(char ***ret_hierarchies) {
         int r;
 
         r = getenv_path_list("SYSTEMD_SYSEXT_HIERARCHIES", ret_hierarchies);
-        if (r < 0)
-                return log_debug_errno(r, "Failed to parse SYSTEMD_SYSEXT_HIERARCHIES environment variable : %m");
+         /* Ignore malformed variables, but propagate OOM */
+        if (r == -ENOMEM)
+                return r;
+        if (r < 0) {
+                log_debug_errno(r, "Failed to parse SYSTEMD_SYSEXT_HIERARCHIES environment variable : %m");
```
Comment (dnicolodi) [inline src/shared/extension-release.c]: Extra space before the ":"?

Comment (bluca) [reply src/shared/extension-release.c]: removed

File: NEWS [hunk 2]
```diff
@@ -20,6 +20,10 @@ CHANGES WITH 248:
           The SYSEXT_LEVEL= field in os-release(5) may be used to specify the
           supported system extension level.
 
+        * A new ExtensionImages= unit setting can be used to apply the same
+          system extension image concept from systemd-sysext to services,
```
Comment (poettering) [inline NEWS]: to services → to the namespaced file hierarchy of specific services?

Comment (bluca) [reply NEWS]: changed

**Pull request merged**